### ISSUE TITLE
all: port to linux/loong64 [freeze exception]

### ISSUE THREAD

**Issue by XiaodongLoong** [golang/go#52825]
I want to request a freeze exception to complete the linux/loong64 port. As far as I know, the feasible reasons are following:
1. As shown in issue #46229, the port status could meet the the [requirements for a new port](https://github.com/golang/go/wiki/PortingPolicy).
2. The influence of porting to loong64 is limited to loong64 only and it should not block 1.19 release cycle. 
3. The golang/loong64 team has been established with four members. I think we have ability to solve the problems in the future.

Thanks! ❤️ 

CC @rsc @golang/release

**Comment by dmitshur** [comment]
Also CC @golang/runtime.

**Comment by rsc** [comment]
How long an exception is needed? We would prefer a time-limited exception. An open-ended exception until the release is probably too risky, even for something that clearly shouldn't affect other ports.

**Comment by XiaodongLoong** [comment]
How about two mounths (in May and June)?  I also want to know the thoughts from runtime team @golang/runtime.

**Comment by aclements** [comment]
Just doing a mid-June check-in. How is the port looking? I know it's in and enabled at this point (🎉). Is there more work to do before closing this issue (and possibly #46229)?

**Comment by xen0n** [comment]
Hi, at this point I don't think the port has any remaining major issue; the Linux ABI change since the port merge [has been addressed](https://go-review.googlesource.com/c/go/+/407694), I compiled some services of my day job, and some CLI tools for my daily workflow, and there was no problem running them.

Some work remains (e.g. regabi support) but these can come later in the go1.20 cycle. I personally think this and #46229 could be closed as finished.

**Comment by XiaodongLoong** [comment]
@aclements The loong64 port has been completed, all CLs in stack have been merged.
I checked the all merged CLs, regabi support is a TODO work for CL https://go-review.googlesource.com/c/go/+/367039, I agree with @xen0n, support this feature in the go1.20 cycle.
There are some issues that have a label named arch-loong64, we will try our best to fix them.
I think the current issue and #46229 could be closed.

Thanks!
